core/roslib/src/roslib/packages.py

```diff
@@ -475,7 +475,6 @@ def find_resource(pkg, resource_name, filter_fn=None, rospack=None):
         returns True if the it matches the desired resource, ``fn(str)``
     :param rospack: `rospkg.RosPack` instance to use
     :returns: lists of matching paths for resource within a given scope, ``[str]``
-    :raises: :exc:`rospkg.ResourceNotFound` If package does not exist 
```

Comment:
Why is this being removed? It looks to me that the `get_path` call in (old) line 495 can raise this exception (https://github.com/ros-infrastructure/rospkg/blob/bffd0aed87fcbf495361289b850be255bd4e1edd/src/rospkg/rospack.py#L203).

Reply:
Please revert this change then.